website_product_configurator/data/config_form_templates.xml

```diff
@@ -60,10 +59,10 @@
                                 <!-- <span class="config_product_price" t-esc="cfg_session.price" t-options="{'widget': 'monetary', 'display_currency': product_tmpl.currency_id}"/> -->
                                 <!-- Bizzappdev start code -->
```

Comment:
What's the purpose of these "Bizzappdev start/end code" comments.
I've seen them in other places, too.

If there's no purpose, could you remove?
If there's a purpose, could you replace them with a more meaningful comment?